packages/sitecore-jss/src/personalize/graphql-personalize-service.ts

```diff
@@ -87,27 +90,36 @@ export class GraphQLPersonalizeService {
       language
     );
 
-    try {
-      const data = await this.graphQLClient.request<PersonalizeQueryResult>(this.query, {
-        siteName: this.config.siteName,
-        itemPath,
-        language,
-      });
+    const cacheKey = this.getCacheKey(itemPath, language);
```

Comment:
We should add `siteName` to the cache key as well.

Reply:
getCacheKey already uses it - it grabs it from the class field.

Reply:
Ah, yep cool.